resources/js/Pages/Results.vue

```diff
@@ -56,15 +92,52 @@
                     value_label: this.labels[mismatch.wikidata_value] || null,
                     ...mismatch
                 }));
-            }
-        }
+            },
+            returnHome(): void {
+                this.$inertia.get( '/' );
+            },
```

Comment:
Instead of forcing inertia back with JavaScript, I think it's best if we use an inertia link and the `as=wikit-button` attribute. See: https://inertiajs.com/links

Reply:
We'll address this in ticket [T290864](https://phabricator.wikimedia.org/T290864)

Reply:
yes, so it can be moved out of here :-)